log/logtest/context.go

```diff
@@ -35,13 +38,26 @@ func WithT(ctx context.Context, t testing.TB) context.Context {
 	// Increase debug level for tests
 	l.SetLevel(logrus.DebugLevel)
 	l.SetOutput(ioutil.Discard)
+	l.SetReportCaller(true)
 
 	// Add testing hook
 	l.AddHook(&testHook{
 		t: t,
 		fmt: &logrus.TextFormatter{
 			DisableColors:   true,
 			TimestampFormat: log.RFC3339NanoFixed,
+			CallerPrettyfier: func(frame *runtime.Frame) (string, string) {
+				file := frame.File
+				if file == "" {
+					file = "???"
+				}
+				line := frame.Line
+				if line == 0 {
+					line = 1
```

Comment:
i'm not sure about this part. should we always add +1 ? or just keep it as is.

Reply:
Or just exclude the information, seems better to return `""` than `???:1`

Reply:
I guess it'll be `:0`

Reply:
@mxpv @dmcgowan updated, use the easiest way. PTAL